storage/scylla_api.c

```diff
@@ -0,0 +1,707 @@
+/*
+ * Copyright (C) 2019 BiiLabs Co., Ltd. and Contributors
+ * All Rights Reserved.
+ * This is free software; you can redistribute it and/or modify it under the
+ * terms of the MIT license. A copy of the license can be found in the file
+ * "LICENSE" at the root of this distribution.
+ */
+#include "scylla_api.h"
+#include <assert.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+
+#define SCYLLA_API_LOGGER "SCYLLA_API"
+
+static logger_id_t logger_id;
+
+void scylla_api_logger_init() { logger_id = logger_helper_enable(SCYLLA_API_LOGGER, LOGGER_DEBUG, true); }
+
+int scylla_api_logger_release() {
+  logger_helper_release(logger_id);
+  if (logger_helper_destroy() != RC_OK) {
+    log_critical(logger_id, "%s.\n", SCYLLA_API_LOGGER);
+    return EXIT_FAILURE;
+  }
+  return 0;
+}
+
+typedef struct Select_Where_ {
+  cass_byte_t* bundle;
+  cass_byte_t* address;
+} Select_Where;
+
+typedef enum { WITH_BUNDLE = 0, WITH_BUNDLE_AND_ADDRESS, NUM_OF_SELECT_METHODS } SelectMethod;
```

Comment:
use `select_method_e` and `select_method_t` as name respectively. Just like what we did in `accelerator/message.h:46`

Reply:
I will revise other structures to follow this.
according to https://github.com/DLTcollab/tangle-accelerator/pull/253#discussion_r317730099
and https://github.com/DLTcollab/tangle-accelerator/pull/253#discussion_r317778188
I am not sure whether I should revise the enumerator.

Reply:
I have answered in that comment